source/install/docker-upgrading.rst

```diff
@@ -1,146 +1,170 @@
 
-Note that this command also refers to |opt.pmm-data| as the value of
-|opt.volumes-from| option. This way, your new version will continue to use the
-existing data.
+This command uses the ``--volumes-from`` option with the value of ``pmm-data``
+so that the new version uses your existing data.
 
-.. warning:: Do not remove the |opt.pmm-data| container when updating,
-	     if you want to keep all collected data.
+.. caution::
 
-Check if the new container is running using |docker.ps|.
+   Do not remove the ``pmm-data`` container when updating, if you want to keep all collected data.
 
-.. include:: ../.res/code/docker.ps.1-5-0.txt
+You can also check that the PMM version has been updated in the PMM Server web interface.
 
-Then, make sure that the |pmm| version has been updated by checking the |pmm-server| web interface.
 
 .. _pmm/docker/backup-container.removing:
+.. _backup-container-removing:
 
-`Removing the backup container <docker-upgrading.html#backup-container-removing>`_
-----------------------------------------------------------------------------------
+*****************************
+Removing the backup container
+*****************************
 
 After you have tried the features of the new version, you may decide to
-continupe using it. The backup container that you have stored
-(:ref:`pmm.deploying.server.docker-container.renaming`) is no longer needed in this
-case.
-
-To remove this backup container, you need the |docker.rm| command:
+continue using it. The backup container that you have stored
+is no longer needed in this case.
 
-.. include:: ../.res/code/docker.rm.pmm-server-backup.txt
+To remove this backup container, use the ``docker rm`` command:
 
-As the parameter to |docker.rm|, supply the tag name of your backup container.
+.. code-block:: bash
 
-.. _pmm/docker/previous-version.restoring:
+   $ docker rm pmm-server-backup
 
-.. rubric:: **Restoring the previous version**
+Here, ``pmm-server-backup`` is the tag name for the backup container.
 
-If, for whatever reason, you decide to keep using the old version, you just need
-to stop and remove the new |opt.pmm-server| container.
+.. _pmm/docker/previous-version.restoring:
 
-.. include:: ../.res/code/docker.stop.pmm-server.rm.txt
+******************************
+Restoring the previous version
+******************************
 
-Now, rename the |opt.pmm-server-backup| to |opt.pmm-server|
-(see :ref:`pmm.deploying.server.docker-container.renaming`) and start it.
+If you decide to keep using the old version, you must
+stop and remove the new ``pmm-server`` container.
 
-.. include:: ../.res/code/docker.start.pmm-server.txt
+.. code-block:: bash
 
-.. warning::
+   $ docker stop pmm-server && docker rm pmm-server
 
-   Do not use the |docker.run| command to start the container. The |docker.run|
-   command creates and then runs a new container.
+Next, rename ``pmm-server-backup`` to ``pmm-server``
+and restart it.
```

Comment:
We give the command for everything else but leave out the docker rename pmm-server-backup pmm-server here.

Reply:
This page probably came from PMM1 and is structured as task sections. Given your comments, I'd prefer to restructure it into numbered steps. It will be simpler. The main heading will still apply but the subheadings will go, replaced with numbers and commentary.